modules/xfem/test/tests/moving_interface/verification/1D_rz_homog1mat.i

```diff
@@ -0,0 +1,187 @@
+#   Thermal conductivity is a single, constant value at all points in the system.
+# Results:
+#   The temperature at the left boundary (x=1) exhibits the largest difference
+#   between the FEM/Moose solution and XFEM results. We present the XFEM results
+#   at this location with 10 digits of precision:
+#     Time    Expected Temperature    XFEM Calculated Temperature
+#      0.2                  440         440
+#      0.4                  480         480.0008118
+#      0.6                  520         520.0038529
+#      0.8                  560         560.0089177
+#      1.0                  600         600.0133344
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ #
+
+[GlobalParams]
+  order = FIRST
+  family = LAGRANGE
+[]
+
+[Problem]
+  coord_type = RZ
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 4
+  ny = 1
+  xmin = 1.0
+  xmax = 2.0
+  ymin = 0.0
+  ymax = 0.5
+  elem_type = QUAD4
+[]
+
+[XFEM]
+  qrule = moment_fitting
+  output_cut_plane = true
+[]
+
+[UserObjects]
+  [./level_set_cut_uo]
+    type = LevelSetCutUserObject
+    level_set_var = ls
+    heal_always = true
+  [../]
+[]
+
+[Variables]
+  [./u]
+  [../]
+[]
+
+[AuxVariables]
+  [./ls]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[Kernels]
+  [./heat_cond]
+    type = MatDiffusion
+    variable = u
+    D_name = 'diffusion_coefficient'
+  [../]
+  [./vol_heat_src]
+    type = BodyForce
+    variable = u
+    function = src_func
+  [../]
+  [./mat_time_deriv]
+    type = TestMatTimeDerivative
+    variable = u
+    mat_prop_value = rhoCp
+  [../]
+[]
+
+[AuxKernels]
+  [./ls_function]
+    type = FunctionAux
+    variable = ls
+    function = ls_func
+  [../]
+[]
+
+[Constraints]
+  [./xfem_constraint]
+    type = XFEMSingleVariableConstraint
+    variable = u
+    geometric_cut_userobject = 'level_set_cut_uo'
+    use_penalty = true
+    alpha = 1e5
+  [../]
+[]
+
+[Functions]
+  [./src_func]
+    type = ParsedFunction
+    value = '10*(-200*x+400) + 200*1.5*t/x'
+  [../]
+  [./neumann_func]
+    type = ParsedFunction
+    value = '1.5*200*t'
+  [../]
+  [./ls_func]
+    type = ParsedFunction
+    value = '2.04 - x - 0.2*t'
+  [../]
+[]
+
+[Materials]
+  [./mat_time_deriv_prop]
+    type = GenericConstantMaterial
+    prop_names = 'rhoCp'
+    prop_values = 10
+  [../]
+  [./therm_cond_prop]
+    type = GenericConstantMaterial
+    prop_names = 'diffusion_coefficient'
+    prop_values = 1.5
+  [../]
+[]
+
+[BCs]
+  [./left_u]
+    type = FunctionNeumannBC
+    variable = u
+    boundary = 'left'
+    function = neumann_func
+  [../]
+  [./right_u]
+    type = DirichletBC
+    variable = u
+    boundary = 'right'
+    value = 400
+  [../]
+[]
+
+[ICs]
+  [./u_ic]
+    type = ConstantIC
+    value = 400
+    variable = u
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  solve_type = 'PJFNK'
+  petsc_options_iname = '-pc_type'
+  petsc_options_value = 'lu'
```

Comment:
It will only run in serial with these options. Can you try running with the default petsc options (just delete these two petsc_options lines) and see if it still works? This goes for all of these problems.

Reply:
It runs fine with the default petsc options. I will delete these lines from all tests.